.dockerignore

```diff
@@ -0,0 +1,8 @@
+logs/*
+data/avvisi/*
+module/*.pyc
+config/settings.yaml
```

Comment:
If even the config/settings.yaml is ignored, how do you transfer the configuration on the container? By entering it with a shell?

Reply:
The same here. We will mount the configuration file on run.